sdcm/cluster.py

```diff
@@ -3125,12 +3125,15 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster', node_prefix='nod
 
         # NOTE: following is needed in case of K8S where we init multiple DB clusters first
         #       and only then we add nodes to it calling code in parallel.
+        azs = len(self.params.get('availability_zone').split(","))
```

Comment:
As I wrote below, I expect it to have problems with EKS backend.

Reply:
this is done to add nodes to each rack uniformly, so currently all the nodes in k8s are in the same rack?

Reply:
> this is done to add nodes to each rack uniformly, so currently all the nodes in k8s are in the same rack?

Yes.